### PR TITLE
Update mccs.yml

### PR DESCRIPTION
**Pull request by Toxblh**
Add USB-C ports
Source https://github.com/alin23/Lunar/blob/master/Lunar/DDC/DDC.swift
Checked on DELL U4021QW

### CODE REVIEW

File: db/data/mccs.yml [hunk 1]
```diff
@@ -1908,7 +1908,13 @@ vcp_features:
   - value: 0x12
     name: HDMI 2
     desc: Digital Video (TMDS) 4
-  - value: ">=0x13"
```
Comment (arcnmx) [inline db/data/mccs.yml:1911]: `>=0x13 && <0x19` will still need to be marked reserved

File: db/data/mccs.yml [hunk 2]
```diff
@@ -1908,7 +1908,13 @@ vcp_features:
...
+  - value: 0x19
+    name: USB-C 1
+    desc: USB-C / Thunderbolt 1
```
Comment (arcnmx) [inline db/data/mccs.yml:1913]: `=0x1A` still reserved too!

File: db/data/mccs.yml [hunk 3]
```diff
@@ -1908,7 +1908,13 @@ vcp_features:
...
+  - value: 0x1B
+    name: USB-C 2
+    desc: USB-C / Thunderbolt 2
+  - value: ">=0x1B"
```
Comment (arcnmx) [inline db/data/mccs.yml]: `>0x1B` or `>=0x1C`

**Review by arcnmx — CHANGES_REQUESTED** [review]

**Comment by arcnmx** [comment]
So I think it just needs to be something like `>=0x1C || =0x1A || (>=0x13 && <=0x18)`?

**Comment by Toxblh** [comment]
Yeah, question. Better cover all gaps in single value, or like I did in last commit?

**Comment by Toxblh** [comment]
I think better to combo in single. Agree with you.

**Comment by Toxblh** [comment]
@arcnmx Could you pls review?

File: db/data/mccs.yml [hunk 4]
```diff
@@ -1908,7 +1908,13 @@ vcp_features:
...
     name: HDMI 2
     desc: Digital Video (TMDS) 4
-  - value: ">=0x13"
+  - value: 0x19
+    name: USB-C 1
+    desc: USB-C / Thunderbolt 1
+  - value: 0x1B
+    name: USB-C 2
+    desc: USB-C / Thunderbolt 2
+  - value: ("=>0x13" && "<=0x18") || "=0x1A" || ">=0x1C"
```
Comment (arcnmx) [inline db/data/mccs.yml:1917]: I'm pretty sure this needs to be a single yaml string, quotes like this will throw off the parser.

**Review by arcnmx — CHANGES_REQUESTED** [review]
Sorry, I've been recovering and busy. I've got the [CI working again](https://github.com/arcnmx/mccs-rs/actions), and moved an old incomplete WIP commit to another branch, if you wouldn't mind rebasing onto the new master?

**Pull request merged**